modular_skyrat/modules/customization/modules/clothing/under/sashes.dm

```diff
@@ -0,0 +1,192 @@
+/obj/item/clothing/under/bodysash
+	name = "assistant sash"
+	desc = "A simple assistant sash, slung from shoulder to hip."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/sashes.dmi' // item icon
+	worn_icon =  'modular_skyrat/master_files/icons/obj/clothing/sashes_worn.dmi' // mob worn icon
+	icon_state = "assistant" // Inventory Icon
+	body_parts_covered = CHEST // |GROIN|ARMS
+	lefthand_file = 'modular_skyrat/master_files/icons/obj/clothing/sashes_hold_left.dmi'
+	righthand_file = 'modular_skyrat/master_files/icons/obj/clothing/sashes_hold_right.dmi'
+	inhand_icon_state = "sash" // In-hand Icon
+	can_adjust = FALSE
+
+/obj/item/clothing/under/bodysash/Initialize()
+	. = ..()
+	desc = "A simple [name], slung from shoulder to hip."
+
+//Captain
+/obj/item/clothing/under/bodysash/captain
+	name = "captain's sash"
+	icon_state = "captain"
+
+//Security
+/obj/item/clothing/under/bodysash/hos
+	name = "head of security's sash"
+	icon_state = "hos"
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
+
+/obj/item/clothing/under/bodysash/warden
+	name = "warden's sash"
+	icon_state = "warden"
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
+
+/obj/item/clothing/under/bodysash/security
+	name = "security's sash"
+	icon_state = "security"
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
+
+/obj/item/clothing/under/bodysash/detective
+	name = "detective's sash"
+	icon_state = "detective"
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
+
+/obj/item/clothing/under/bodysash/brigdoc
+	name = "brig physician's sash"
+	icon_state = "brigdoc"
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
+
+/obj/item/clothing/under/bodysash/deputy
+	name = "deputy's sash"
+	icon_state = "deputy"
+	sensor_mode = SENSOR_COORDS
+	random_sensor = FALSE
```

Comment:
What's the point of keeping a sash for a job you don't have? Also, shouldn't you add more sashes for skyrat-exclusive jobs?